data/pages/nostr.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Nostr
+summary: "Fix the Money, Fix the Web"
+image: /static/images/projects/nostr.png
+layout: PageLayout
+---
+
+Our [mission] is to maintain a sustainable ecosystem of funding for free and
+open-source projects and contributors, especially bitcoin-related projects and
+protocols that help bitcoin flourish. We believe that nostr is such a protocol.
+Like bitcoin, it is an open protocol that has the potential to revolutionize the
+way we think about individual sovereignty and liberty.
+
+To aid the various development efforts that are being made in the nostr
+ecosystem, we created [The Nostr Fund](https://opensats.org/projects/nostr) to
+provide funding for open-source projects. If you would like to join us in
+helping with these efforts, please reach out to us or [donate to the
+fund](https://opensats.org/projects/nostr).
+
+If you are an open-source developer working on notes and other stuff transmitted
+by relays, don't hesitate to [apply for funding][apply].
+
+# Nostr Design
+
+In addition to providing funding to projects, we offer to help with design work
+if your open-source project needs it.
+
+Part of these efforts is [nostrdesign.org](https://nostrdesign.org/), a
+comprehensive resource offering product design considerations, design
+principles, and best practices that will help developers solve the unique user
+experience challenges that nostr poses.
+
+In addition to nostrdesign.org, two of our grantees are dedicating time to help
+projects that need help with design and general UI/UX improvements. 
+
+The first one is `@karnage`, who brings a lot of experience and expertise from
+the world of bitcoin product design. Karnage was involved in the development of
+multiple nostr clients and is the brain behind *Hey Nostr*[^fn-hey] and
+*Nostr Apps*.[^fn-apps]
+
+The second one is `@dtonon`, who is
+actively involved in the design of major clients, most notably Gossip and
+Coracle, as well as various tools such as the static content gateway that is
+part of *nostr.com*.[^fn-nostr-com]
+
+[^fn-hey]: [heynostr.com](https://www.heynostr.com/)
+[^fn-apps]: [nostrapps.com](https://www.nostrapps.com/)
+[^fn-nostr-com]: [nostr.com/npub10pensat...](https://nostr.com/npub10pensatlcfwktnvjjw2dtem38n6rvw8g6fv73h84cuacxn4c28eqyfn34f)
```

Comment:
@dergigi why does nostr.com have a footnote that points to the OpenSats profile?

Reply:
I wanted to show off `njump` and pointing to the OpenSats profile felt natural. Feel free to change to whatever you think is best!